css-fonts-and-colors/styles.css

```diff
@@ -0,0 +1,51 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Poppins, sans-serif;
+  background-color: rgba(187, 206, 216, 0.965);
+}
```

Comment:
We only want the `background-color` to be applied to the `section`. Right now the whole page will be the light blue. What if we want the page to be one and this "early access" section to be an accent?